pkg/controller/perconaxtradbcluster/perconaxtradbcluster_controller.go

```diff
@@ -333,6 +339,48 @@ func (r *ReconcilePerconaXtraDBCluster) deploy(cr *api.PerconaXtraDBCluster) err
 	return nil
 }
 
+func (r *ReconcilePerconaXtraDBCluster) checkSecret(cr *api.PerconaXtraDBCluster, namespace string) error {
+	secretObj := corev1.Secret{}
+	err := r.client.Get(context.TODO(),
+		types.NamespacedName{
+			Namespace: namespace,
+			Name:      cr.Spec.PXC.SSLSecretName,
+		},
+		&secretObj)
+	if err != nil {
+		return err
+	}
+	if len(secretObj.Name) == 0 && cr.Spec.PXC.AllowUnsafeConfig == false {
+		var (
+			issuerKind = "ClusterIssuer"
+			issuerName = cr.Spec.PXC.SSLSecretName + "-ca"
+		)
+		issuer := cm.ClusterIssuer{}
+		issuer.Namespace = namespace
+		issuer.Kind = issuerKind
+		issuer.Name = issuerName
+		issuer.Spec.SelfSigned = &cm.SelfSignedIssuer{}
+		err := r.client.Create(context.TODO(), &issuer)
+		if err != nil {
+			return err
+		}
```

Comment:
From the comments for the previous PR:
There always will be an "AlreadyExists" error after the issuer has been created. See https://github.com/percona/percona-xtradb-cluster-operator/blob/c3149a7010f1a476ba6122b6326ddb473517c482/pkg/controller/perconaxtradbcluster/perconaxtradbcluster_controller.go#L227-L230 for example of Create's error handling.